instance/tests/integration/factories/instance.py

```diff
@@ -64,3 +64,16 @@ def create(cls, *args, **kwargs):
     configuration_version = 'integration-ginkgo'
     edx_platform_repository_url = 'https://github.com/open-craft/edx-platform.git'
     edx_platform_commit = 'opencraft-release/ginkgo.1'
+
+
+class OpenEdXInstanceFactoryWithSimpleTheme(OpenEdXInstanceFactory):
+    """
+    Factory for OpenEdXInstance including simple_theme. It requires different branches
+    because simple_theme wasn't merged upstream yet.
+    """
+    # This branch is based on 'integration-ginkgo' and has simple_theme role on top
+    configuration_source_repo_url = 'https://github.com/open-craft/configuration.git'
+    configuration_version = 'integration-ginkgo-with-simpletheme'
```

Comment:
@clemente The simpletheme role looks pretty good to me, so we can just merge it into `integration-ginkgo` and remove this temporary `OpenEdXInstanceFactoryWithSimpleTheme` class.

Can you please open a PR against `integration-ginkgo` branch to add simpletheme role? I'll approve it and we can merge, then remove this class.

Reply:
PR: https://github.com/open-craft/configuration/pull/26
I'll squash changes again after doing this change.